spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AppRegistryController.java

```diff
@@ -428,24 +428,22 @@ else if (!StringUtils.isEmpty(apps)) {
 	 * {@link org.springframework.cloud.deployer.resource.support.DelegatingResourceLoader}.
 	 */
 	private void prefetchMetadata(List<AppRegistration> appRegistrations) {
-		forkJoinPool.execute(() -> {
-			appRegistrations.stream().filter(r -> r.getMetadataUri() != null).parallel().forEach(r -> {
-				logger.info("Eagerly fetching {}", r.getMetadataUri());
-				try {
-					this.appRegistryService.getAppMetadataResource(r);
-				}
-				catch (Exception e) {
-					logger.warn("Could not fetch " + r.getMetadataUri(), e);
-				}
-			});
-		});
+		forkJoinPool.execute(() -> appRegistrations.stream().filter(r -> r.getMetadataUri() != null).parallel().forEach(r -> {
+			logger.info("Eagerly fetching {}", r.getMetadataUri());
+			try {
+				this.appRegistryService.getAppMetadataResource(r);
+			}
+			catch (Exception e) {
+				logger.warn("Could not fetch {}", r.getMetadataUri(), e);
+			}
+		}));
```

Comment:
Hope this block is just formatting change from and complies with their expected eclipse-formatting file?

Reply:
Changed formatting here.